[2025-03-20T01:10:54.324+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-20T01:10:54.369+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: welcome_dag.print_random_quote scheduled__2025-03-18T23:00:00+00:00 [queued]>
[2025-03-20T01:10:54.398+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: welcome_dag.print_random_quote scheduled__2025-03-18T23:00:00+00:00 [queued]>
[2025-03-20T01:10:54.407+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-20T01:10:54.478+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): print_random_quote> on 2025-03-18 23:00:00+00:00
[2025-03-20T01:10:54.491+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'welcome_dag', 'print_random_quote', 'scheduled__2025-03-18T23:00:00+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/welcome_dag.py', '--cfg-path', '/tmp/tmp0sbul3gk']
[2025-03-20T01:10:54.495+0000] {standard_task_runner.py:105} INFO - Job 111: Subtask print_random_quote
[2025-03-20T01:10:54.499+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=219) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-20T01:10:54.500+0000] {standard_task_runner.py:72} INFO - Started process 237 to run task
[2025-03-20T01:10:54.641+0000] {task_command.py:467} INFO - Running <TaskInstance: welcome_dag.print_random_quote scheduled__2025-03-18T23:00:00+00:00 [running]> on host e3789c96a6c5
[2025-03-20T01:10:54.958+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='welcome_dag' AIRFLOW_CTX_TASK_ID='print_random_quote' AIRFLOW_CTX_EXECUTION_DATE='2025-03-18T23:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-18T23:00:00+00:00'
[2025-03-20T01:10:54.961+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-20T01:10:55.296+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097 InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.quotable.io'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
[2025-03-20T01:10:55.341+0000] {logging_mixin.py:190} INFO - Quote of the day: "The best thing about the future is that it only comes one day at a time."
[2025-03-20T01:10:55.343+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-20T01:10:55.396+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-20T01:10:55.399+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=welcome_dag, task_id=print_random_quote, run_id=scheduled__2025-03-18T23:00:00+00:00, execution_date=20250318T230000, start_date=20250320T011054, end_date=20250320T011055
[2025-03-20T01:10:55.641+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-20T01:10:55.681+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-20T01:10:55.686+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
